xline/src/storage/execute_error.rs

```diff
@@ -26,6 +26,11 @@ impl ExecuteError {
         Self::KvError("key not found".to_owned())
     }
 
+    /// Key not found
```

Comment:
Please modify the comment. And why should we consider an invalid request error as a kind of KvError?

Reply:
the code is now refactored, please refer to #398